### ISSUE TITLE
Add platforms and versioning to GitHub repository (readme?) and to the in-app Terms and Conditions

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#16874]
### Description

We are receiving bug reports for Linux Mint, NixOS, Fedora, CentOS, i guess we need to showcase which linux is currently working for users along with other platforms. Initial motivation is to highlight Ubuntu 22.04 as the only Linux distro we support.

Platfroms and versions:

**Linux**: Ubuntu 20.04
**MacOS**: Intel and Apple Silicone chips, latest MacOS
**Windows**: Intel chip only now (arm processor is "working" but we dont build natively for it yet), Windows 11

Here's a draft block we could include: 

**Supported platforms and versions**
To ensure the best user experience, our application is optimized for specific platforms and versions. Use outside these environments may result in limited functionality or unsupported performance.

**Desktop:**
Linux: Officially supported on Ubuntu 20.04. Other distributions, such as Linux Mint, NixOS, Fedora, and CentOS, are not supported.
macOS: Supported on devices with Intel and Apple Silicon chips, running the latest macOS version.
Windows: Supported on Intel-based devices running Windows 11. ARM processor devices are experimental, with no native build currently available.

**Comment by anastasiyaig** [comment]
@jrainville can we include that in 2.32 maybe?

**Comment by alexjba** [comment]
I'm afraid this will discourage people to try out Status on other linux flavors and also to log issues and start discussions when they run Status on other linux flavors. Our goal, as I see it, is to encourage people to try Status on all linux flavors and provide feedback on GH, discord and support page. Most bugs and issues will probably be common to all linux flavors, with just a few outliers.

As a result, maybe it's worth rephrasing to be explicit in what we're actively doing in terms of support and what's our expectations:

- There's a continuous effort to support the lowest Ubuntu LTS version.
- We're actively testing Ubuntu 22.04 (I would suggest replacing the `official support` term with this point here).
- We're expecting Status to run on most linux flavors using glibc 2.31 or newer
- We're encouraging people running Status on any OS to contribute by logging the issues they face on GH and to get in touch for support.

**Comment by anastasiyaig** [comment]
ah sure, thats perfect @alexjba thanks for suggesting this!

**Comment by jrainville** [comment]
I agree that it's important to be clearer with users what we support. I also think we need to find a way to convey that some versions are 100% supported, ie they should **not** have high impact bugs, and that some other versions, while they should work, we cannot guarantee that they will work perfectly.

I'll put it in 2.33 as I donMt think it's a super big issue, since we didn't have those since the beginning of the public betas. Also, the readme is version agnostic, so we can update it anytime and it will show on the github main page.

We can try to find the right wording to make it sound good and encouraging.

Maybe @jorge-campo and his team can help 😃
Anyway, we should probably update the user docs too

**Comment by anastasiyaig** [comment]
perfect thanks a lot :)

**Comment by jorge-campo** [comment]
@Fabiomorais87, the points discussed on this issue are worth mentioning in the Linux download article. Please review the information here and update the article accordingly. We might need to add a new H2 to clarify this information.

@anastasiyaig , thank you for bringing up this topic! Only a minor thing: I believe we should avoid using the term "official support" because we (Status) don't have an official support channel or support level agreement. So far, CCs only provide _best-effort support_ to users with questions or issues.

CC/ @alexjba , @jrainville

**Comment by anastasiyaig** [comment]
@jorge-campo sure, i was not putting so much of deep sense in this definition :) i just wanted to say that we test and develop mostly on linux 20.04 and we have Lukas with Fedora, thats it

**Comment by anastasiyaig** [comment]
I created https://www.notion.so/Desktop-Platforms-and-OS-support-19f8f96fb65c80ff8fadd11bc8437f13 as per Volo's request. If this task is no longer needed , feel free to close it

**Comment by jrainville** [comment]
I added a "Supported versions" section to the readme in this PR: https://github.com/status-im/status-desktop/pull/17938
